app/grandchallenge/algorithms/templates/algorithms/algorithm_card_body.html

```diff
@@ -28,7 +28,7 @@
         <a class="badge badge-info above-stretched-link p-1 mr-1"
            href="{{ publication.identifier.url }}"
            title="{{ publication.title }}">
-            <i class="fas fa-file fa-fw"></i>&nbsp;Article

```

Comment:
OK - one `&nbsp;` is fine - this says "These things belong together on the same line, don't break them like a normal space". It was your use of two of them to force a certain spacing that was problematic. Please keep your PRs simple. They should do one thing, not touch 31 files to add a link to another page.

Reply:
Roger. I try to balance 'while I have my hands in this code ...' with keeping PRs focused. I oversteped here.